.gitignore

```diff
@@ -58,8 +58,6 @@ python/*.egg
 python/.eggs/
 python/.tox
 python/build/
-python/google/protobuf/compiler/
-python/google/protobuf/util/
```

Comment:
Maybe we should keep these, but add things like `!python/google/protobuf/compiler/__init__.py`.

Reply:
Note that the `compiler` and `util` directories only contain generated `*_pb2.py` files besides the `__init__.py` files. The `*_pb2.py` files are already ignored in L56. 

Why do you want to keep the two lines?

Reply:
As I'm guessing there is a reason for them. Even though I'm not entirely sure what it is.

Reply:
Me neither. git blame doesn't help much in this case ...

Reply:
They are in here for generated directories. It is reasonable to remove them as now __init__ files are added